zuul/connection/github.py

```diff
@@ -566,6 +566,10 @@ def getPullBySha(self, sha):
 
         if len(pulls) > 1:
             raise Exception('Multiple pulls found with head sha %s' % sha)
+            for pull in pulls:
+                self.commentPull(pull.owner, pull.project, pull.pr_number,
+                                 'Multiple pulls found with head sha %s. '
+                                 'Please amend your change.' % sha)
```

Comment:
We probably want this msg to be configurable in the same way it is for merge failures, test success, etc?